website/blog/2023-11-1-happy-birthday-to-us/item.md

```diff
@@ -0,0 +1,113 @@
+
+We can say that our growing pains haven’t been too dreadful. Sure, there was the one month when we went back and forth between three tools for tracking work tasks, but… In general, there’s nothing that a good conversation and some testing can’t solve.
+
+> Perhaps the biggest growing pain is the rest of the team learning how to prevent the founder from working himself into exhaustion. ;-)
+
+### A founder’s brain and heart
+
+Our team at authentik has a shared love of building things, and that shapes both how we work together and also our product, even how we communicate with our community.
+
+An interesting offset to our shared love of building is the shared sense of humility, of which we get daily doses from Jens.
+
+> To build boldly yet with humility is what sets some founders apart from others.
+
+The tone and espirit of the company is one reason it’s so meaningful to celebrate our 1-year birthday; we can happily celebrate a hard year of doing things with full, enthusiastic engagement. At authentik, nerdiness is embraced, technical curiousity flourishs, and transparency is a big part of our nature. Speaking of how we communicate with our community, our Discord forum is (in addition to GitHub) an important place where transparency matters. For example, we recently asked our community what they preferred for a release cycle. Based on the answers, we lengthened the release time from from monthly to every two or three months.
+
+Moving from a role of solo creator of an open source project, to being primary maintainer of a popular, growing project, to suddenly being CTO of a company based on that project is a quite a transition. A natural question we wanted to ask Jens is “What’s been the hardest thing about building a company?” His answers:
+
+-   “Recognizing and accepting that you don’t get to work on only what you want to, 100% of time… “
+-   “Learning to delegate, learning to let go a bit, trusting others to do it in their way, in the right spirit. Especially letting others get into the code… I’ve learned that instead of saying ‘I would not have done it this way’, I instead measure the success of the change itself.”
+
+### What’s up next?
+
+Going forward, we want to of course keep our focus on building features and supporting authentication protocols that our users want, but we have also identified several specific goals for this coming year:
```

Comment:
Suggestion: The "... of course ..." kinda breaks the flow.  It works verbally, but read silently it feels a bit like a hiccup.